docker-for-mac/release-notes.md

```diff
@@ -467,7 +467,7 @@ events or unexpected unmounts.
   - Linux Kernel 4.9.60
 
 * Bug fixes and minor changes
-  - Fix login into private repository with certificate issue. (docker/for-mac#2201)
+  - Fix login into private repository with certificate issue. [https://github.com/docker/for-mac/issues/2201](docker/for-mac#2201)
```

Comment:
@gtardif this looks like a bug. Can you fix this going forward so that the link is created correctly?